arlo-client/src/components/DataEntry/BallotAudit.tsx

```diff
@@ -127,34 +122,48 @@ const BallotAuditContest = ({
     })
   }
 
-  // Since the radio button values must be strings, not full objects,
+  // Since the checkbox button values must be strings, not full objects,
   // we condense our data model for interpretations to strings:
   // - For buttons representing interpretation VOTE (one for each
   // contest choice), the value is interpretation.choiceId
```

Comment:
I think this line is slightly out of date now

Reply:
@jonahkagan Actually, it looks like we can remove the `value` field altogether.

Reply:
And upon things breaking because I was using that to define the testid used by the tests to find the actual element they need to interact with, I added it back in partially.